submissions/rocket-silo/rocket-silo.md

```diff
@@ -0,0 +1,21 @@
+
+### Modules and the Silo
+
+You can fit 12 beacons around 3x3 entities, and 16 around oil refineries. You can fit 20 around a silo. 2 beacons with 2 speed module 3s in each is the same as adding a second machine.
+So one silo can do the work of 13 silos, side-by-side. When I mention costs, they are the relevant ores in vanilla without productivity modules in the crafting chain, and I am also using advanced oil processing. Each beacon costs 135 iron, 110 copper, 20 coal, and 411 oil which is quite a bit for an early-game player but nothing if tier 3 modules are automated (https://factoriolab.github.io/list?p=beacon*1&v=1). That's because any tier 3 module is 1,083 iron, 2,000 copper, 190 coal, and 4,129 crude oil (https://factoriolab.github.io/list?p=speed-module-3*1&v=1). While the calculator mentions speed modules specifically, the other modules are equivalent. So 4 tier 3 modules and 2 beacons costs 4,552 iron, 8,270 copper, 800 coal, and 17,334 crude oil (https://factoriolab.github.io/list?p=speed-module-3*4_beacon*2&v=1). That's a lot of raw materials. With certain optimizations, like those shown here https://factorio.com/blog/post/fff-351, its possible to get more out of your module, but the cost still stands. Normally, this cost greatly outmatches the machine. However, there is 1 exception to this rule, and that's the silo. The silo costs a whopping 12,220 iron, 8,600 copper, 437 coal, 11,080 crude oil and 1,000 stone (https://factoriolab.github.io/list?p=rocket-silo*1&v=1). So, its cheaper than making another silo to place down. Less spaghetti involved, too!
+
+### Productivity Bonus Interlude
+
+When I say crafting operation, I refer to the main bar within the assembling-machine UI filling up to completion. For the sake of explanation, productivity is basically a multiplier on your output for your given input. The way this works is by taking 100 inputs and multiplying that by the percentage bonus from the modules within the machine. So, for example, take 4 productivity module 3s inside some copper cable assembling machine 3.
```

Comment:
I think it would be good to link to this article for anyone wanting more info about productivity modules: https://alt-f4.blog/ALTF4-12/#complexity-corner-productivity-modules-thekool